02/styles/global.css

```diff
@@ -4,6 +4,18 @@
     box-sizing: border-box;
 }
 
+body {
+    display: grid;
+    grid-template-areas: 
+    "top"
+    "middle"
+    "bottom";
```

Comment:
Często jest mylone przez mentee to, że nazwy dot. template-area są uważane za klasy CSS - warto na to zwracać uwagę ;P